dm/Dockerfile

```diff
@@ -18,7 +17,7 @@ RUN apk update && apk add bash
 
 RUN make dm-worker dm-master dmctl
 
-FROM alpine:3.15
+FROM rockylinux:9.0
```

Comment:
Why should this change be made? Should the relevant dockerfile in engine be changed to this as well?

Reply:
Yes, engine will replace alpine with rockylinux too.